packages/components/src/slot-fill2/index.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
What about the set of tests for the slot implementation? I don't see them ported.

I even copied them and tried to run locally but they all fail.

Reply:
I moved `slot-fill2` into `slot-fill` and replaced the current `bubblesVirtually` implementation with it. I had to remove only one assertion that was testing the component instance, but the rest is working fine. I'll add more tests later.